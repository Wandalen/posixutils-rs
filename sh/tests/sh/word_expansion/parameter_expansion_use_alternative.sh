VAR=value NULL=''

echo ${VAR:+alternative}
echo ${NULL:+alternative}
echo ${UNSET:+alternative}
echo $VAR $NULL $UNSET

echo ${VAR+alternative}
echo ${NULL+alternative}
echo ${UNSET+alternative}
echo $VAR $NULL $UNSET