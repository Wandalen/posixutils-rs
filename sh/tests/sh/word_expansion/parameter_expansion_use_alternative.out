alternative


value
alternative
alternative

value
